docs/chain-details/introduction.md

```diff
@@ -0,0 +1,38 @@
+### Introduction
+
+Current traditional payment infrastructure and existing blockchain-powered payment networks have failed to provide a wide-spread, easy-to-integrate and fast settlement of cryptocurrency in the real world. In particular, the key limitations of existing payment network infrastructures are that they:
+
+- Do not integrate with cryptocurrencies systematically;
+- Do not allow a way for either customers or merchants to reconcile payment figures in a trustless way;
+- Are vulnerable due to being the central point of failure;
+- Are expensive to operate; and
+- Give low limits on cryptocurrency spending by default.
+
+The development of blockchain technology and cryptocurrencies represents a cryptography and security breakthrough as significant as that created by the internet in the 1990s. This technology, however, is still at a very nascent stage; in order to generate mass adoption, it will therefore be necessary to find compelling real-life use cases that can appeal to an audience larger than the small group of industry professionals and experts.
+
+## Crypto.com Chain 
+
+We believe that enabling cryptocurrency spending in the real world will constitute an adoption catalyst. Current traditional payment institutions and existing blockchains have failed to provide a secure, scalable and decentralized solution to support cryptocurrency payment. The key limitations of existing blockchain-powered payment networks are that they are:
+
+- Too complex to setup and use;
+- Not friendly to crypto first timers; and
+- Rarely supported beyond their own blockchain;
+
+Accordingly, we propose Crypto.com Chain, the next generation decentralized mobile payment protocol, the most efficient and secure way to pay and be paid in crypto, anywhere, any crypto at little to no cost. Crypto.com Chain will deliver on its vision by developing innovative technology components and processes catered specifically to cryptocurrency payment, while leveraging proven blockchain technology structural design elements.
```

Comment:
is the re-pivoted version still mobile payment-focused? probably sync with business white paper what it says?

Reply:
[Aligned](https://github.com/crypto-com/chain-docs/pull/38/commits/89c4e43c18b989be0b84687afa780187dfe932ee#diff-37b5ab464a89b7d9971b077c76530f03e2f9ef228dc5652967508ac0f745b1c5R9) with the business white paper.